apps/nextjs/src/app/api/chat/route.test.ts

```diff
@@ -1,21 +1,40 @@
 import { Aila } from "@oakai/aila";
 import { MockLLMService } from "@oakai/aila/src/core/llm/MockLLMService";
 import { MockCategoriser } from "@oakai/aila/src/features/categorisation/categorisers/MockCategoriser";
+import { AilaPersistedChat } from "@oakai/aila/src/protocol/schema";
 import { mockTracer } from "@oakai/core/src/tracing/mockTracer";
+import { prisma } from "@oakai/db";
 import { NextRequest } from "next/server";
 
 import { consumeStream } from "../../../utils/testHelpers/consumeStream";
 import { expectTracingSpan } from "../../../utils/testHelpers/tracing";
 import { handleChatPostRequest } from "./chatHandler";
 import { Config } from "./config";
 
+const chatId = "test-chat-id";
+const userId = "test-user-id";
+
+const persistedChat: AilaPersistedChat = {
+  id: chatId,
+  userId,
+  messages: [],
+  isShared: false,
+  path: "",
+  title: "",
+  lessonPlan: {},
+  createdAt: new Date(),
+};
+
 describe("Chat API Route", () => {
   let testConfig: Config;
   let mockLLMService: MockLLMService;
   let mockChatCategoriser: MockCategoriser;
   beforeEach(() => {
     mockTracer.reset();
     jest.clearAllMocks();
+    jest
```

Comment:
Not sure about this - if we don't want persistence in this test, we should pass usePersistence: false